sunpy/net/dataretriever/attrs/gbm.py

```diff
@@ -0,0 +1,15 @@
+from sunpy.net.attr import SimpleAttr
+
+__all__ = ['Detector', 'Datatype']
+
+
+class Detector(SimpleAttr):
+    """
+    Detector number for FERMI GBM
+    """
+
+
+class Datatype(SimpleAttr):
```

Comment:
Would using the existing `PhysObs` attr be applicable here?

Reply:
yeah I guess the only thing is is that its a detector number rather than a physical observable? But maybe that doesn't matter

Should I change it to use vso.Physobs instead?

Reply:
@Cadair I actually think the vso.Resolution attr makes the most sense! It is a difference in both the time and energy resolution between the two - that make sense?

Reply:
Works for me if you think it's the right move :smile:

Reply:
groovy - changed to that now :)